managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedgerConfig.java

```diff
@@ -77,6 +77,7 @@
     private int newEntriesCheckDelayInMillis = 10;
     private Clock clock = Clock.systemUTC();
     private ManagedLedgerInterceptor managedLedgerInterceptor;
+    private Map<String, String> topicMetadata;
```

Comment:
In my understanding, this `topicMetadata ` is for meta info of PartitionedTopic. Does it have anything to do with ManagedLedger？

Reply:
Maybe we'd better don't add topic-related concepts in the ManagedLedger package, why not add this in the `PersistentTopic`?

Reply:
Yes，we may don't need to add this field. 
Before, I store this metadata to the ledger properties in ledger#openLedgerComplete. But consideration of atomic, add this field.  We will discuss this later. This pr will take some time to be merged.

Reply:
@Jason918 @gaoran10

Reply:
Changed `topicMetadata` to `properties` which may cause confused

Reply:
Discuss with @codelipenghui , add this field to ledger config can decrease zk op.